test/formatLocation.test.js

```diff
@@ -1,8 +1,10 @@
-var should = require("should");
-var formatLocation = require("../lib/formatLocation");
+"use strict";
```

Comment:
You're right. For some reason I thought this was the `webpack/webpack.js.org` repository. Nevermind.